backend/src/main/java/ca/bc/gov/backendstartapi/enums/GeneticClassEnum.java

```diff
@@ -0,0 +1,21 @@
+package ca.bc.gov.backendstartapi.enums;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+/** This enumeration represents a specie genetic class. */
+@Schema(description = "This object represents a specie genetic class.")
```

Comment:
I found a definition for _specie_ as a nonstandard spelling of _species_, but in most dictionaries its only defined in the financial sense.